packages/vulcan-forms/lib/components/Form.jsx

```diff
@@ -168,7 +169,7 @@ class SmartForm extends Component {
   /*
 
   Get a list of all editable fields
-  
+
```

Comment:
It'd be great to only have the actual code changes in the PR if possible

Reply:
done, sorry about the automatic linting